.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
we shouldn't need to update this file, these things are handled by create-react-app. please remove.

Reply:
I do not remember what do i suppose to remove on it file . can you send me the link of estlintrc and prettier format that you sent earlier ?

Reply:
shouldn't need it at all, create-react-app works some magic and it's not visible to the user.